packages/discovery-react-components/src/components/SearchFacets/components/FacetsGroups/MultiSelectFacetsGroup.tsx

```diff
@@ -118,7 +118,9 @@ export const MultiSelectFacetsGroup: FC<MultiSelectFacetsGroupProps> = ({
           tempSelectedFacets.find(({ selectedFacetKey, checked }) => {
             if (selectedFacetKey === facetText) {
               facetSelected = !!checked;
+              return true;
             }
+            return false;
```

Comment:
Why are there return statements needed? Doesn't seem like the result of the `find` is used anywhere...

Reply:
1. This change was made due to linter saying we weren't using `find()` correctly.
2. But now I see that this should instead be a `forEach`, since we're not using the result.

Reply:
Actually, changing to use `every`, since that can be short-circuited once we have a match.